### PR TITLE
Feat: video lib contexts

### PR DESCRIPTION
**Pull request by tewarimohit955**
Features Added :-
- Added Home Page
- Created contexts
- Used API in context

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]

File: src/Pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from "react";
+import "../Styles/Home.css";
+import Header from "../Components/Header";
+import Sidebar from "../Components/Sidebar";
+import VideoCard from "../Components/VideosCard";
+
+
+const Home = () => {
+	
+
```
Comment (arjunbharti774) [inline src/Pages/Home.js:10]: You can remove extra spacing here

File: src/Styles/Sidebar.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.side-drawer {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: flex-start;
+	width: 250px;
+	/* height: 100vh; */
```
Comment (arjunbharti774) [inline src/Styles/Sidebar.css:7]: avoid unnecessary comments

File: src/index.js [hunk 3]
```diff
@@ -3,13 +3,25 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
+import { WatchLaterProvider } from "./Context/WatchLaterContext";
+import { LikedVideosProvider } from "./Context/LikedVideosContext";
+import { VideoProvider } from "./Context/videoContext";
```
Comment (arjunbharti774) [inline src/index.js:9]: you can create an `index.js` file in the context folder and import all of them in this single file. Importing them here and writing these context imports in a single line make the code look clean!

**Review by arjunbharti774 — APPROVED** [review]
Nice approach followed for file structure. Contexts are nicely wriiten.
Just suggested a minor change. You can go ahead!

**Pull request closed without merging**